Add new NoPopUps project

djaus2 opened the pull request:
IoT-Core does not support the PopUp class.
This project demonstrates how to have a [Yes] [Cancel] Popup panel with a message with the app awaiting the user response and responding accordingly..

djaus2 committed:
NoPopUps

Added NoPopUps example project

djaus2 committed:
Added NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Update NoPopUps.md

djaus2 committed:
Improved the simulated PopUp appearance

The popup now appears to popup infront of some UI elements.

djaus2 committed:
Update README.md

msftclas commented:
Hi **@djaus2**, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;

djaus2 committed:
Update README.md

djaus2 committed:
"Domunatation for this sample" link in ReadMe.md now points to my NoPopUps.md in my copy of the repo

ie At 
https://github.com/djaus2/samples/blob/develop/NoPopups/NoPopUps.md

djaus2 committed:
The "Documentation for this sample" link in README.md now points to NoPopUps.md in my branch

So I can maintain it.
ie
https://github.com/djaus2/samples/blob/develop/NoPopups/NoPopUps.md
